addons/core/translations/form/en-us.yaml

```diff
@@ -51,6 +51,9 @@ worker_last_seen:
   help: Last time the worker reported status.
 worker_address:
   help: URL and port for this worker.
+worker_tags:
```

Comment:
I think we already have a translation for [worker tags](https://github.com/hashicorp/boundary-ui/blob/main/addons/core/translations/resources/en-us.yaml#L481) and the help text.

Maybe we should check in with what text we should stick with and be consistent.